src/dra/627C4.c

```diff
@@ -592,7 +592,7 @@ s32 HandleSaveMenu(s32 arg0) {
                 return 2;
             }
             if (g_pads[0].tapped & CONFIRM) {
-                PlaySfx(SE_UI_CONFIRM);
+                PlaySfx(SFX_UI_SELECT);
```

Comment:
In my mind confirm is pressing X and select is pressing up/down/left/right. If this is select what are you going to call up/down/left/right?

Reply:
I was planning on having the `0x67B` sound cues labeled as `SFX_UI_MOVE` since it involves the D-pad.

I might just revert back to confirm because it does make a little more sense.  One of the reasons why I changed it to select is that I'm trying to find a good name for the Start Button confirm bang at the title screen, but that cue is used for other cues besides UI like door sounds and Dracula monster stomps.

Reply:
Reverting now...